src/components/WorkflowsExplorer/Timeline/useTaskData.ts

```diff
@@ -0,0 +1,184 @@
+import { Row, Step, TaskStatus } from '../../../types';
+import {
+  getStepStatus,
+  makeTasksForStep,
+  timepointsOfTasks,
+} from './taskdataUtils';
+
+//
+// useTaskData hook is responsible of fetching all step and task data for given run. It automatically
+// fetches, and receives all realtime updates from server.
+//
+// Row data is grouped by steps
+//
+
+export type StepRowData = {
```

Comment:
Initially there were Step and Task rows, and we decided to use only Task, right?
Do we still need this StepRowData definition?

Reply:
No we don't. I should go again through the Timeline component and clean it further at some point